Throwings.cpp

```diff
@@ -0,0 +1,54 @@
+﻿#include "Throwings.h"
+#include "ScoreAndLife.h"
+
+void Throwings::update(ScoreAndLife& score) {
+    if (destroyed == true) {
+        return;
+    }
+    if (this->is_food == true) {
+        Vector2f pos = food.getPosition();
+        if (pos.y > 480.0f - 26.0f) {
+            speed = 0;
+            destroyed = true;
+
+            clock.restart();
+        }
+        food.move(0.0f, 1.0f * speed);
+
+    }
+    else if (this->is_food == false) {
+        Vector2f pos = ddong.getPosition();
+        if (pos.y > 480.0f - 26.0f) {
+            speed = 0;
+            destroyed = true;
+
+            clock.restart();
+        }
+        ddong.move(0.0f, 1.0f * speed);
+    }
```

Comment:
```suggestion
    if (this->is_food) {
        Vector2f pos = food.getPosition();
        if (pos.y > 480.0f - 26.0f) {
            speed = 0;
            destroyed = true;

            clock.restart();
        }
        food.move(0.0f, 1.0f * speed);

    }
    else{
        Vector2f pos = ddong.getPosition();
        if (pos.y > 480.0f - 26.0f) {
            speed = 0;
            destroyed = true;

            clock.restart();
        }
        ddong.move(0.0f, 1.0f * speed);
    }
```
로 수정해주실 수 있을 것 같습니다!

Reply:
앗 혹시 c++은 `if(this->is_food) `작동이 되지 않나요..? 제가 테스트는 해보지 않고 단 댓글입니다.

Reply:
실행 잘 되네용❗수정해두겠습니당:)